### PR TITLE
Update version v3.1.0 to v3.1.1

### PR DESCRIPTION
**Pull request by serenashe**
/assign @Patrick-LuoYu @Felixnoo

### CODE REVIEW

File: content/en/common/kubernetes-versions.md [hunk 1]
```diff
@@ -5,6 +5,6 @@ _build:
 
 | Installation Tool | KubeSphere version | Supported Kubernetes versions                                |
 | ----------------- | ------------------ | ------------------------------------------------------------ |
-| KubeKey           | v3.1.0             | v1.17.0, v1.17.4, v1.17.5, v1.17.6, v1.17.7, v1.17.8, v1.17.9, v1.18.3, v1.18.5, v1.18.6, v1.18.8, v1.19.0, v1.19.8, v1.19.9, v1.20.4 |
-| ks-installer      | v3.1.0             | v1.17.x, v1.18.x, v1.19.x, v1.20.x                           |
+| KubeKey           | v3.1.1             | v1.17.0, v1.17.4, v1.17.5, v1.17.6, v1.17.7, v1.17.8, v1.17.9, v1.18.3, v1.18.5, v1.18.6, v1.18.8, v1.19.0, v1.19.8, v1.19.9, v1.20.4 |
```
Comment (Felixnoo) [inline content/en/common/kubernetes-versions.md:8]: Please add `v1.20.6` to `Supported Kubernetes versions`.
Xnip2021-07-21_15-18-43

File: content/zh/common/kubernetes-versions.md [hunk 2]
```diff
@@ -5,5 +5,5 @@ _build:
 
 | 安装工具     | KubeSphere 版本 | 支持的 Kubernetes 版本                                       |
 | ------------ | --------------- | ------------------------------------------------------------ |
-| KubeKey      | v3.1.0          | v1.17.0, v1.17.4, v1.17.5, v1.17.6, v1.17.7, v1.17.8, v1.17.9, v1.18.3, v1.18.5, v1.18.6, v1.18.8, v1.19.0, v1.19.8, v1.19.9, v1.20.4 |
-| ks-installer | v3.1.0          | v1.17.x, v1.18.x, v1.19.x, v1.20.x                           |
+| KubeKey      | v3.1.1          | v1.17.0, v1.17.4, v1.17.5, v1.17.6, v1.17.7, v1.17.8, v1.17.9, v1.18.3, v1.18.5, v1.18.6, v1.18.8, v1.19.0, v1.19.8, v1.19.9, v1.20.4 |
```
Comment (Felixnoo) [inline content/zh/common/kubernetes-versions.md:8]: Please add `v1.20.6` to `Supported Kubernetes versions`.
Xnip2021-07-21_15-18-43

**Review by Felixnoo — CHANGES_REQUESTED** [review]

File: content/en/common/kubernetes-versions.md (hunk 1 above)
Comment (serenashe) [reply content/en/common/kubernetes-versions.md]: Added.

File: content/zh/common/kubernetes-versions.md (hunk 2 above)
Comment (serenashe) [reply content/zh/common/kubernetes-versions.md]: Added. Thanks!

**Pull request merged**